x/evm/statedb/journal.go

```diff
@@ -24,21 +24,21 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// journalEntry is a modification entry in the state change journal that can be
-// reverted on demand.
-type journalEntry interface {
-	// revert undoes the changes introduced by this journal entry.
-	revert(*StateDB)
+// JournalEntry is a modification entry in the state change journal that can be
+// Reverted on demand.
+type JournalEntry interface {
+	// Revert undoes the changes introduced by this journal entry.
+	Revert(*StateDB)
 
-	// dirtied returns the Ethereum address modified by this journal entry.
-	dirtied() *common.Address
+	// Dirtied returns the Ethereum address modified by this journal entry.
+	Dirtied() *common.Address
 }
 
 // journal contains the list of state modifications applied since the last state
-// commit. These are tracked to be able to be reverted in the case of an execution
+// commit. These are tracked to be able to be Reverted in the case of an execution
```

Comment:
No need for capital R. Probably refactoring did it automatically. Same for some other comments